flang/test/Driver/arch-specific-libdir-rpath.f95

```diff
@@ -0,0 +1,32 @@
+// REQUIRES: x86-registered-target
```

Comment:
Please could you use fortran style comments (starting with `!`)

Reply:
Done